### ISSUE TITLE
Is VOLINDEX.xml necessary?

### ISSUE THREAD

**Issue by matthewmarks-bbc** [IMFTool/IMFTool#27]
I know it's just a small extra file that can be deleted without consequence, but does anyone use this any more? Its namespace is ...429-9/2007, but ST 429-9:2014 says:

"The VolumeIndex structure is not used. Both the VolumeIndex global element (but not the VolumeIndex element of the ChuckTypetype) and the VolumeIndexType global type are deprecated accordingly.
Note: Earlier versions of this specification included support for distributing Asset Maps across multiple storage volumes."

**Comment by IMFTool** [comment]
I agree VOLINDEX.xml is deprecated. But unless users report issues, I'll keep it for the sake of backwards compatibility.
